### PR TITLE
Use box-shadow instead of filter: drop-shadow

### PR DESCRIPTION
**Pull request by taipeicoder**
Related to #76821

## Proposed Changes

This PR fixes an issue with the `filter: drop-shadow()` rendering in Firefox. Might be related to this issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1772176.

See screenshot for reference:
237686816-fd662a9f-aa84-408e-8fe1-1db2df40bb5b

## Testing Instructions

* Use Firefox.
* Head to /setup/site-setup?siteSlug=${site_slug}.
* Continue until you land on the Onboarding Design Picker.
* Ensure that the gray boxes are no longer there.
* Also test that the Theme Showcase works as before.

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [x] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: packages/design-picker/src/components/theme-card/style.scss [hunk 1]
```diff
@@ -50,9 +50,12 @@ $theme-card-info-margin-top: 16px;
 	position: relative;
 }
 
+.theme-card__image-wrapper {
+	box-shadow: 0 15px 25px rgba(0, 0, 0, 0.05);
+}
+
 .theme-card__image-container {
 	border: 1px solid rgba(0, 0, 0, 0.1);
-	filter: drop-shadow(0 15px 25px rgba(0, 0, 0, 0.05));
```
Comment (arthur791004) [inline packages/design-picker/src/components/theme-card/style.scss:55]: Question: Why not just replace it with `box-shadow`?

**Review by arthur791004 — APPROVED** [review]
Tested and looks good to me! I cannot see the weird shadow with this change 👍

File: packages/design-picker/src/components/theme-card/style.scss (hunk 1 above)
Comment (taipeicoder) [reply packages/design-picker/src/components/theme-card/style.scss:55]: Hmmm when I tried before, the `box-shadow` it was truncated, but it doesn't seem to be the case anymore. Good catch!

**Review by arthur791004 — APPROVED** [review]
Still looks good 👍

**Review by simison — APPROVED** [review]
Works when testing `/setup/free` — thanks for the quick fix!

**Review by okmttdhr — APPROVED** [review]
LGTM! Thank you ✅

**Pull request merged**